Makefile

```diff
@@ -4,6 +4,7 @@ MODULES ?= ${TARGETS:=.pp.bz2}
 # Point SHAREDIR to DATADIR by default to not break existing users
 DATADIR ?= /usr/share
 SHAREDIR ?= ${DATADIR}
+CONFDIR ?= /etc
```

Comment:
@rhatdan `SYSCONFDIR` would be preferable.

Reply:
Changed.